h2o-core/src/main/java/water/fvec/persist/PersistUtils.java

```diff
@@ -31,7 +31,7 @@ static String sanitizeUri(String uri) {
         }
     }
 
-    static void write(URI uri, Writer w) {
+    public static void write(URI uri, Writer w) {
```

Comment:
```suggestion
    public static void write(URI uri, Writer w) throws RuntimeException {
```

Reply:
Yes, it's only a matter of preference - therefore a suggestion. I found it quite nice to declare exceptions thrown, even if Java doesn't force us to do so. It's your call, no big deal. It might as well stay as-is.